ci-operator/step-registry/openshift/upgrade/qe/test/canary/openshift-upgrade-qe-test-canary-chain.yaml

```diff
@@ -0,0 +1,25 @@
+chain:
+  as: openshift-upgrade-qe-test-canary
+  steps:
+  - ref: idp-htpasswd
+  - ref: cucushift-pre
+  - ref: openshift-extended-upgrade-pre
+  - ref: cucushift-upgrade-prepare
+  - ref: cucushift-upgrade-prehealthcheck
+  - ref: mco-conf-day2-create-custom-mcp
+  - ref: cucushift-upgrade-eus-pause-worker-mcp
+  - ref: cucushift-ota-preupgrade
+  - ref: cucushift-upgrade-toimage
+  - ref: cucushift-upgrade-eus-unpause-worker-mcp
+  - ref: mco-conf-day2-destroy-custom-mcp
+  - ref: cucushift-upgrade-rhel-worker
+  - ref: cucushift-upgrade-healthcheck
+  - ref: cucushift-ota-postupgrade
```

Comment:
I do not think we have OTA case enabled on the canary upgrade, so we can drop it here.